### DISCUSSION TITLE
Using Persona cards

### DISCUSSION THREAD

**Discussion by EbiAlcero** [microsoft-search/pnp-modern-search#1841] [Q&A] [answered]
Hi everybody,

I want to have the same feature of Persona cards (show on mouse hover) in other views such as Details list, custom,...
I can add the persona cards tags in the custom code but I do not know how to add hover functionality to it, how can we use javascript or events in the "Result template"?
It would be great if somebody could help me with this.

**Comment by wobba** [comment, marked as answer]
Make sure loading of graph toolkit is checked and then try something like we use in the people template https://github.com/microsoft-search/pnp-modern-search/blob/main/search-parts/src/layouts/results/people/people.html#L42

**Reply by EbiAlcero** [reply]
Thank you so much for helping me on this :D
